### ISSUE TITLE
[BUG] Crash on rhode

### ISSUE THREAD

**Issue by xowny** [Alovoa/alovoa-expo#82]
**Check if you're in the correct repository**

This repository only covers bugs of app.alovoa.com and iOS/Android apps only. Go to [alovoa](https://github.com/Alovoa/alovoa) for reporting bugs on app.alovoa.com and the backend.

**Describe the bug**

Always crashes in the final screen after uploading photos and choosing preferences.

**To Reproduce**

1. Exactly after the screen that says "start getting likes" it always crashes, no matter what.

**Expected behavior**

**Screenshots**

** Device (please complete the following information):**
 - OS: [Android]
 - OS version: 14
 - Device: [Motorola G52 - Rhode]

**Native Application:**

 - 1.12.0
 - F-droid

**Web application**

 - Browser [Firefox]
 - Browser version: 130.0.1

**Additional context**

- Using a degoogled lineage os rom.
- Crashlog:
```
type: crash
osVersion: motorola/rhode_g/rhode:13/T2SRS33.72-22-4-7/f93e7e-118bc5:user/release-keys
package: com.alovoa.expo:34
process: com.alovoa.expo
processUptime: 29511 + 37 ms
installer: com.machiav3lli.fdroid

com.facebook.react.common.JavascriptException: Invariant Violation: scrollToIndex out of range: requested index 6 is out of 0 to 5, js engine: hermes, stack:
invariant@1:76502
scrollToIndex@1:508356
scrollToIndex@1:496413
anonymous@1:2097229
scrollToIndex@1:2097461
moveFlatlistNext@1:2231256
_performTransitionSideEffects@1:402801
_receiveSignal@1:402389
onResponderRelease@1:401219
invokeGuardedCallbackImpl@1:323008
invokeGuardedCallback@1:323065
invokeGuardedCallbackAndCatchFirstError@1:323097
executeDispatch@1:323229
executeDispatchesAndReleaseTopLevel@1:327461
forEachAccumulated@1:324727
anonymous@1:327813
batchedUpdatesImpl@1:385804
batchedUpdates@1:327381
_receiveRootNodeIDEvent@1:327654
receiveTouches@1:380711
__callFunction@1:84454
anonymous@1:82950
__guard@1:83888
callFunctionReturnFlushedQueue@1:82908

	at com.facebook.react.modules.core.ExceptionsManagerModule.reportException(ExceptionsManagerModule.java:65)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.facebook.react.bridge.JavaMethodWrapper.invoke(JavaMethodWrapper.java:372)
	at com.facebook.react.bridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:188)
	at com.facebook.jni.NativeRunnable.run(Native Method)
	at android.os.Handler.handleCallback(Handler.java:959)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:27)
	at android.os.Looper.loopOnce(Looper.java:232)
	at android.os.Looper.loop(Looper.java:317)
	at com.facebook.react.bridge.queue.MessageQueueThreadImpl$4.run(MessageQueueThreadImpl.java:228)
	at java.lang.Thread.run(Thread.java:1012)
```

**Comment by Nonononoki** [comment]
Have you tried tapping on the button in the center of the last page?

**Comment by Nonononoki** [comment]
Should be fixed now with 1.13.0
